src/ragas/testset/testset_generator.py

```diff
@@ -392,8 +390,9 @@ def generate(
             if is_valid_question:
                 context = self._generate_context(question, text_chunk)
                 answer = self._generate_answer(question, context)
-                samples.append(
-                    DataRow(question.split("\n"), context, answer, evolve_type)
+                samples.extend(
+                    DataRow(qstn, [ctx], ans, evolve_type)
```

Comment:
`ground_truth` should be `List[str]`  and not `str`

Reply:
shouldn't we change that to `str`?

Reply:
ground_truth is expecting List[str]